asapdiscovery-alchemy/asapdiscovery/alchemy/schema/fec.py

```diff
@@ -60,6 +61,105 @@ def to_solvent_component(self) -> gufe.SolventComponent:
         return gufe.SolventComponent(**self.dict(exclude={"type"}))
 
 
+class AdaptiveSettings(_SchemaBase):
+    """
+    A settings class to encode settings for adaptive settings. These were recommended by OpenFE.
+    """
+
+    type: Literal["AdaptiveSettings"] = "AdaptiveSettings"
+    adaptive_sampling: bool = Field(
```

Comment:
This should go away, instantiating the class and attatching to `FreeEnergyCalculationFactory` should signal use of adaptive sampling.